src/lib.rs

```diff
@@ -467,6 +473,44 @@ fn apply_patches(target: &str, inner: &Path) {
         .unwrap();
 }
 
+fn apply_patches_aarch64_apple_darwin(target: &str, inner: &Path) {
+    if target != "aarch64-apple-darwin" {
+        return;
+    }
+
+    // Apply build system changes to allow configuring and building
+    // for Apple's ARM64 platform.
+    // https://github.com/openssl/openssl/pull/12369
+
+    let mut buf = String::new();
+    let path = inner.join("Configurations/10-main.conf");
+    File::open(&path).unwrap().read_to_string(&mut buf).unwrap();
+
+    assert!(
+        !buf.contains("darwin64-arm64-cc"),
+        "{} already contains instructions for aarch64-apple-darwin",
+        path.display(),
+    );
+
+    const PATCH: &'static str = r#"
```

Comment:
`'static` has not been required on constants [since 1.17.0](https://github.com/rust-lang/rust/blob/master/RELEASES.md#language-28)

Reply:
Yup. Again, just being overly cautious because I wasn't aware of the crate's MSRV requirements.